.prettierrc

```diff
@@ -1,3 +1,5 @@
 {
-  "singleQuote": true
+    "tabWidth": 2,
+    "printWidth": 140,
+    "singleQuote": true
```

Comment:
i am a single quote non-enjoyer

Reply:
It was already true. I didn't even change it...